src/test/java/uk/gov/hmcts/ccd/domain/types/DocumentValidatorTest.java

```diff
@@ -130,6 +136,18 @@ public void shouldValidateDocumentWithValidUrlAndDomain() {
         assertEquals(validDocumentUrlResult.toString(), 0, validDocumentUrlResult.size());
     }
 
+    @Test
+    public void shouldValidateDocumentWithValidEmHrsUrlAndDomain() {
```

Comment:
Please add some tests for some near match URLs with only some parts mismatching.